### PR TITLE
[zh]Update content/zh/docs/reference/kubectl/conventions.md

### PR DESCRIPTION
**Pull request by zyy19981018**

### CODE REVIEW

**Comment by tengqm** [comment]
Rather than having contributors randomly pick a page to sync, please check the issues #33474, #33475, #33476, #33477, #33478, #33479 and sign on tasks there. It would help save your time and avoid conflicts when working on different pages.

File: content/zh/docs/reference/kubectl/conventions.md [hunk 1]
```diff
@@ -43,6 +43,23 @@ For a stable output in a script:
 * 完全限定版本。例如 `jobs.v1.batch/myjob`。这将确保 kubectl 不会使用其默认版本，该版本会随着时间的推移而更改。
 * 不要依赖上下文、首选项或其他隐式状态。
 
+<!--
+## Subresources
+-->
+## 子资源
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/conventions.md:49]:
```suggestion
## 子资源    {#subresources}
```

File: content/zh/docs/reference/kubectl/conventions.md [hunk 2]
```diff
@@ -43,6 +43,23 @@ For a stable output in a script:
...
+<!--
+* You can use the `--subresource` alpha flag for kubectl commands like `get`, `patch`,
+`edit` and `replace` to fetch and update subresources for all resources that
+support them. Currently, only the `status` and `scale` subresources are supported.
+* The API contract against a subresource is identical to a full resource. While updating the
+`status` subresource to a new value, keep in mind that the subresource could be potentially
+reconciled by a controller to a different value.
+-->
+
+* 你可以将 `--subresource` alpha 标志用于 kubectl 命令，例如 `get`、`patch`、`edit` 和 `replace` 来获取和更新所有资源的子资源支持他们。 目前，仅支持 `status` 和 `scale` 子资源。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/conventions.md:60]: 参考[样式指南](https://kubernetes.io/zh/docs/contribute/localization_zh/)，适当折行。

```suggestion
* 你可以将 `--subresource` alpha 标志用于 kubectl 命令，例如 `get`、`patch`、`edit` 和 `replace`
来获取和更新所有支持子资源的资源的子资源。 目前，仅支持 `status` 和 `scale` 子资源。
```

File: content/zh/docs/reference/kubectl/conventions.md [hunk 3]
```diff
@@ -43,6 +43,23 @@ For a stable output in a script:
...
+* 针对子资源的 API 协定与完整资源相同。在更新`status` 子资源到一个新值，请记住，子资源可能是潜在的由控制器调和为不同的值。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/conventions.md:61]:
```suggestion
* 针对子资源的 API 协定与完整资源相同。在更新`status` 子资源为一个新值时，请记住，
子资源可能会被控制器调和为不同的值。
```

File: content/zh/docs/reference/kubectl/conventions.md [hunk 4]
```diff
@@ -57,8 +57,10 @@ support them. Currently, only the `status` and `scale` subresources are supporte
 reconciled by a controller to a different value.
 -->
 
-* 你可以将 `--subresource` alpha 标志用于 kubectl 命令，例如 `get`、`patch`、`edit` 和 `replace` 来获取和更新所有资源的子资源支持他们。 目前，仅支持 `status` 和 `scale` 子资源。
-* 针对子资源的 API 协定与完整资源相同。在更新`status` 子资源到一个新值，请记住，子资源可能是潜在的由控制器调和为不同的值。
+* 你可以将 `--subresource` alpha 标志用于 kubectl 命令，例如 `get`、`patch`、`edit` 和 `replace`
+来获取和更新所有支持子资源的资源的子资源。 目前，仅支持 `status` 和 `scale` 子资源。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/conventions.md:61]:
```suggestion
  来获取和更新所有支持子资源的资源的子资源。 目前，仅支持 `status` 和 `scale` 子资源。
```

File: content/zh/docs/reference/kubectl/conventions.md [hunk 5]
```diff
@@ -57,8 +57,10 @@ support them. Currently, only the `status` and `scale` subresources are supporte
...
+* 针对子资源的 API 协定与完整资源相同。在更新`status` 子资源到一个新值，请记住，
+子资源可能是潜在的由控制器调和为不同的值。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/conventions.md:63]:
```suggestion
  子资源可能是潜在的由控制器调和为不同的值。
```

**Comment by tengqm** [comment]
I can squash the commits for you, but next time please do keep your PR with a single commit.

/label tide/merge-method-squash

File: content/zh/docs/reference/kubectl/conventions.md [hunk 6]
```diff
@@ -43,6 +43,25 @@ For a stable output in a script:
...
+`edit` and `replace` to fetch and update subresources for all resources that
+support them. Currently, only the `status` and `scale` subresources are supported.
+* The API contract against a subresource is identical to a full resource. While updating the
+`status` subresource to a new value, keep in mind that the subresource could be potentially
+reconciled by a controller to a different value.
+-->
+
+* 你可以将 `--subresource` alpha 标志用于 kubectl 命令，例如 `get`、`patch`、`edit` 和 `replace`
+  来获取和更新所有支持子资源的资源的子资源。 目前，仅支持 `status` 和 `scale` 子资源。
+* 针对子资源的 API 协定与完整资源相同。在更新`status` 子资源到一个新值，请记住，
```
Comment (mengjiao-liu) [inline content/zh/docs/reference/kubectl/conventions.md:62]: 没有完成 https://github.com/kubernetes/website/pull/33483#discussion_r865697456 建议的修改
```suggestion
* 针对子资源的 API 协定与完整资源相同。在更新`status` 子资源为一个新值时，请记住，
```

**Pull request merged**